src/compiler/crystal/tools/playground/views/layout.html.ecr

```diff
@@ -14,7 +14,7 @@
   </head>
   <body class="grey lighten-3 grey-text text-darken-2">
     <header>
-      <div class="navbar-fixed">
+      <div class="navbar">
```

Comment:
I guess you prefer non fixed navbar. But other than that, any other reason to change this?

Probably becuase when zooming the navbar is also zoomed.
The css is pretty straight forward and changing the font-size should affect mostly everything in the main area.
Maybe there should be a presenter mode with even bigger font size...